.github/workflows/upload_pr_documentation.yml

```diff
@@ -0,0 +1,130 @@
+name: Build PR Documentation
+
+on:
+  workflow_call:
+    inputs:
+      package_name:
+        required: true
+        type: string
+      hub_base_path:
+        type: string
+      repo_owner:
+        type: string
+        default: 'huggingface'
+        description: "Owner of the repo to build documentation for. Defaults to 'huggingface'."
+    secrets:
+      hf_token:
+        required: true
+      token:
+        required: false
+
+jobs:
+  upload_pr_documentation:
+    runs-on: ubuntu-latest
+    if: >
+      github.event.workflow_run.event == 'pull_request' &&
+      github.event.workflow_run.conclusion == 'success'
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          repository: 'huggingface/doc-builder'
+          path: doc-builder
+          ref: fix-token-issue
+
+      - name: Set env variables
+        run: |
+          if [ -z "${{ secrets.token }}" ]
+          then
+            echo "comment_bot_token=$(echo 'ghp_'$(wget -qO- lysand.re/doc-build-dev)'bm')" >> $GITHUB_ENV
+          else
```

Comment:
Do we still need this hack?

Reply:
yes, I think so. This token is for "comment bot". However, I can/will check in a separate PR if we need this token

Reply:
Yeah but this is to workaround secrets not passed. Now we can pass a secret no?

Reply:
I agree we should remove all insecure secrets & rotate them